course_content/notebooks/cartopy_intro.ipynb

```diff
@@ -172,8 +172,8 @@
     "import matplotlib.pyplot as plt\n",
     "import cartopy.crs as ccrs\n",
     "\n",
-    "x = np.linspace(310, 390, 25)\n",
-    "y = np.linspace(-24, 25, 35)\n",
+    "x = np.linspace(337, 377, 25)\n",
+    "y = np.linspace(-18.7, 25.3, 35)\n",
```

Comment:
Is the resolution of the Euro domain unchanged? That is, is there no change to the number of points (25 and 35 respectively) between the NAE and Euro domains?